lib/openstax/aws/stack.rb

```diff
@@ -325,6 +325,20 @@ def self.format_hash_as_tag_parameters(tags)
       tags.map{|tag| {key: tag.key, value: tag.value}}
     end
 
+    def self.list_stacks(region: "us-east-1")
+      client = Aws::CloudFormation::Client.new(region: region)
+
+      results = []
+
+      client.list_stacks().each do |response|
+        response.stack_summaries.each do |stack|
+          results.push(stack.stack_name)
+        end
+      end
+
+      return results
```

Comment:
I think `map` is the go-to function for creating an array based on an another array
```ruby
client.list_stacks.map { |response| response.map ...}.flatten
```
 maybe?